xprof/tally.hpp

```diff
@@ -9,6 +9,9 @@
 #include <tuple>
 #include <unordered_map>
 #include <vector>
+#include <string>
+#include <array>
+#include <cassert>
```

Comment:
I don't like transitive insert. If you need them in the `cpp` please include in the cpp.